src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/Networking/UvWriteReq.cs

```diff
@@ -31,6 +31,13 @@ public UvWriteReq(ILibuvTrace logger) : base(logger)
         {
         }
 
+        public override void Init(LibuvThread thread)
+        {
+            Init(thread.Loop);
+
+            base.Init(thread);
+        }
+
         public void Init(UvLoopHandle loop)
```

Comment:
Yea thats how I started it but I don't want to refactor those tests as part of this change. It would need to be internal and it's still accessible from all of the code in there.